flavour/gcloud/sample/docker-compose.yaml

```diff
@@ -0,0 +1,13 @@
+version: "3"
+services:
+    cli:
+        volumes:
+            - "/gcloud_key:/path/to/directory/with/gcloud_key.json"
```

Comment:
This has the wrong order. you need to swap the two pathes.
And maybe it could be wise to replace the dot with an underscore or maybe one really gives the path to the final file instead of to the folder

Reply:
Done